source/JsMaterialX/JsMaterialXCore/JsUnit.cpp

```diff
@@ -16,23 +16,28 @@ extern "C"
         ems::class_<mx::UnitConverter>("UnitConverter")
             .smart_ptr<std::shared_ptr<mx::UnitConverter>>("UnitConverter")
             .smart_ptr<std::shared_ptr<const mx::UnitConverter>>("UnitConverter")
-            .function("convert_float", ems::select_overload<float(float, const std::string&, const std::string&) const>(&mx::UnitConverter::convert))
-            .function("convert_vector2", ems::select_overload<mx::Vector2(const mx::Vector2&, const std::string&, const std::string&) const>(&mx::UnitConverter::convert))
-            .function("convert_vector3", ems::select_overload<mx::Vector3(const mx::Vector3&, const std::string&, const std::string&) const>(&mx::UnitConverter::convert))
-            .function("convert_vector4", ems::select_overload<mx::Vector4(const mx::Vector4&, const std::string&, const std::string&) const>(&mx::UnitConverter::convert))
+            .function("convertFloat", ems::select_overload<float(float, const std::string&, const std::string&) const>(&mx::UnitConverter::convert))
```

Comment:
Why not follow the pattern here with having a `define BIND_CONVERT ...` up front?

Reply:
I thought it not worth it for the two classes and few functions